src/components/App/AppComponent.js

```diff
@@ -20,6 +21,9 @@ export class AppComponent extends PureComponent {
       case PAGES_ENUM.NOTES:
         return <NotesPageContainer />;
 
+      case PAGES_ENUM.NOTE_DETAILS:
+        return <NoteDetailsPage />;
```

Comment:
The PR says "Delete redundant code", but it adds note details page. I'm confused.

Reply:
Please start merge from bottom to top
![gittree](https://user-images.githubusercontent.com/45026388/96812765-4064f300-1457-11eb-8145-23acd964554d.png)

Starting from **addnote-page**, then **movelogout**, etc.

Reply:
Ok, I've reviewed addnote https://github.com/DarthVanger/NeuralNotes/pull/159